File: Programmers/LV0/Go/src/각도기.go
package main

import "fmt"

func solution_각도기(angle int) int {
	if angle > 0 && angle < 90 {
		return 1
	} else if angle == 90 {
		return 2
	} else if angle > 90 && angle < 180 {
		return 3
	} else {
		return 4
	}
}

func main() {
	fmt.Println(solution_각도기(30))
}

File: Programmers/LV0/Java/각도기.java
public class 각도기 {
    public static void main(String[] args) {
        // example
        Solution solution = new Solution();
        int result = solution.solution(30);
        System.out.println(result);
    }
}

class Solution {
    public int solution(int angle) {
        int answer = 0;
        if (angle > 0 && angle < 90) {
            answer = 1;
        } else if (angle == 90) {
            answer = 2;
        } else if (angle > 90 && angle < 180) {
            answer = 3;
        } else {
            answer = 4;
        }
        return answer;
    }
}